wasm-syntax.md

```diff
@@ -173,8 +173,8 @@ When a binary operator is the next instruction, `#eval<T>BinOp` will be called o
  // ---------------------------------------------------------------
 ```
 
-Arithmetic
-----------
+Numeric Operators
```

Comment:
For sanity, shouldn't it also be `### Numeric Operators`?

Reply:
I'm trying to mimic the structure of the WASM spec. Numeric Operators starts on page 63. I think it's best to wait until a bit more of the semantics is finished, then I'll do a re-organizational PR to make it mimic the WASM spec much more.